src/service/gnuradio/examples/picoscope4000a-streaming.grc

```diff
@@ -30,23 +30,46 @@ blocks:
         - Test unit B
         - Test unit C
         - Test unit D
-      name: fair::picoscope::Picoscope4000a<float>
-      sample_rate: 80000
+      sample_rate: 10000
       streaming_mode_poll_rate: 0.000010
+  - name: convertA
+    id: gr::blocks::type::converter::Convert
+    template_args: "float,double"
+  - name: convertB
+    id: gr::blocks::type::converter::Convert
+    template_args: "float,double"
+  - name: convertC
+    id: gr::blocks::type::converter::Convert
+    template_args: "float,double"
+  - name: convertD
+    id: gr::blocks::type::converter::Convert
+    template_args: "float,double"
   - name: sinkA
     id: gr::basic::DataSink
-    template_args: float
+    template_args: double
+    parameters:
+      signal_name: test
```

Comment:
Setting the signal_name explicitly here overrides/hides the PS `channel_names` to signal_name propagation, which this tested also.

Reply:
So you would like at least one of the sinks to not explicitly specify `signal_name`? I can change that. This was while I was not getting any updates at all and was trying to make the picoscope flowgraph as similar to the builtin one as possible.